src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <Article
+      title={article.title}
+      date={article.date}
+      text={article.text}
+    />
```

Comment:
You can just use spread `<Article {...article} />`

Reply:
Thanks! It`s a good idea!